vnext/ReactWindows.sln

```diff
@@ -127,12 +127,12 @@ Global
 		Chakra\Chakra.vcxitems*{c38970c0-5fbf-4d69-90d8-cbac225ae895}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|ARM = Debug|ARM
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
-		Release|ARM = Release|ARM
+		Debug|ARM = Debug|ARM
```

Comment:
There is no actual change on this file. you may rollback change for this file.

Reply:
There is, actually. This defaults the solution's build platform to x64, instead of ARM.